src/main/java/core/basesyntax/db/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.db;
+
+public class FruitTransaction {
+    private String fruit;
+    private int quantity;
+    private String type;
+
+    public FruitTransaction(String fruit, int quantity) {
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
```

Comment:
This constructor only initializes `fruit` and `quantity`, but not `type`. If `type` is a required attribute for a transaction, it should be initialized in all constructors to avoid leaving the object in an inconsistent state.